docs/operations_guide/Flink.md

```diff
@@ -29,6 +29,44 @@ In particular, one must not forget that the Flink connector (when checkpoints ar
 * Commits the offsets to Kafka only during checkpoint - so offsets returned by Kafka almost always will not be correct.
 * Ignore offsets in Kafka when it’s started with the checkpointed state - topic offsets are also saved in the checkpointed state.
 
+#### EXACTLY_ONCE delivery mode
+
+The mode is well described here: [this section of Kafka connector documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/connectors/datastream/kafka/#fault-tolerance).
+In order to configure the mode you need to:
+- Configure the property `components.kafka.config.deliveryGuarantee` to: "EXACTLY_ONCE", e.g.
```

Comment:
We should:
1. Add this also to this table: https://nussknacker.io/documentation/docs/integration/KafkaIntegration/#configuration-for-flink-engine
2. Write down possible values (I think that writing it in this table will be enough) 
3. At the same incompatible parameter check as we do with `idleTimeout` in `LiteKafkaComponentProvider`